documentation/create-official-version.adoc

```diff
@@ -0,0 +1,17 @@
+
+If you consider the status of the curriculum worthy of a new release or release candidate, you have to create a `tag` with Git.
+
+The format for a release candidate tag is as follows: +
+`2021.1-RCX`, where x is either 1 or the current release candidate number increased by one. +
+This tag will create a GitHub _pre-release_. They are [available here](https://github.com/isaqb-org/curriculum-foundation/releases) and provide you with a zip archive that contains the curriculum as PDF and HTML. +
+**ATTENTION:** You have to also append the current RC version to the version stored in the `document.version` file. If you forget to do that, the RC information will NOT be part of the document's version.
+
+The format for a release tag is as follows: +
+`2021.1-revX` where x is either 0 (initial release), or the current revision increased by one. +
+This tag will create a GitHub _Release_. They are [available here](https://github.com/isaqb-org/curriculum-foundation/releases) and provide you with a zip archive that contains the curriculum as PDF and HTML.
+In addition, the released curriculum will be published on GitHub Pages. +
+**ATTENTION:** You have to make sure to define the correct version in the `document.version` file. If you forget to do that, the document's version will be wrong.
```

Comment:
see above.

Reply:
but your proposal still mentions `document.version`?

Reply:
It does not? "See above" refers to my comment in this file above.

Reply:
Sorry for that. Just remove the references to `document.version`everywhere. 😀